includes/rest/class-actors-inbox-controller.php

```diff
@@ -178,7 +178,15 @@ public function create_item( $request ) {
 
 		// phpcs:ignore WordPress.Security.ValidatedSanitizedInput
 		if ( \wp_check_comment_disallowed_list( $activity->to_json( false ), '', '', '', $_SERVER['REMOTE_ADDR'], $_SERVER['HTTP_USER_AGENT'] ?? '' ) ) {
-			Debug::write_log( 'Blocked activity from: ' . $activity->get_actor() );
+			/**
```

Comment:
What about simply changing the `$type` and removing the `else`? This way we could use the actions in both cases?

Reply:
What do you think about using `block` as `$type`?

This is a valid ActivityType, so we would be consistent and it is only used in the C2S spec, so it is normally not meant to go to the inbox but the outbox: https://www.w3.org/TR/activitypub/#block-activity-outbox

Reply:
But either way, I think we can remove the `_activity` at the end.

Reply:
I tried to clearly differentiate this action from the wildcard activity type action. With using `block` I worry that it would be confusing with actual incoming Block activities that should be processed. My preference would be for it to be something completely different, so activity processing can be done reliably.

What if we broke with the `_inbox` namespace and called it `activitypub_disallowed_activity`?

Reply:
This is a bit generic... We could use that if we provide a `$context` to give a hint where it happened?

Reply:
In the Outbox endpoint we have a `acrivitypub_rest_outbox_*` namespece, what about `activitypub_rest_inbox_disallowed`?

Reply:
Fixed in [339ca2f](https://github.com/Automattic/wordpress-activitypub/pull/1620/commits/339ca2f04c34f0ecd41d3fbe31ca13a82d782dfe).